opendatahub-io/data-science-pipelines-operator
# Replaced kfdefs with DataScienceCluster resource and changed default ns to opendatahub

## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/453

## Description of your changes:



Replaced kfdefs with DataScienceCluster resource and changed default ns to opendatahub

## Testing instructions


In a cluster with ODH Operator with 'fast' channel installed/ODH operator version > 2.0, create the following `DataScienceCluster`:

```bash
$ cat <<EOF | oc apply -f -
kind: DataScienceCluster
apiVersion: datasciencecluster.opendatahub.io/v1
metadata:
  name: data-science-pipelines-operator
  namespace: opendatahub
spec:
  components:
    dashboard:
      managementState: Managed
    datasciencepipelines:
      managementState: Managed
      devFlags:
        manifests:
          - uri: https://github.com/hbelmiro/data-science-pipelines-operator/tarball/issue-453
            contextDir: config
            sourcePath: base
EOF
```

The data-science-pipelines-operator should start successfully.

```bash
$ oc get pods -n opendatahub   
NAME                                                              READY   STATUS    RESTARTS   AGE
data-science-pipelines-operator-controller-manager-66b79b6lzrrz   1/1     Running   0          29s
odh-dashboard-f9c79d89b-k44pf                                     2/2     Running   0          33s
odh-dashboard-f9c79d89b-wz4xb                                     2/2     Running   0          33s
```

## Checklist
